user-guide/tactical-handbook/tactical-handbook.md

```diff
@@ -0,0 +1,350 @@
+
+Alternatively, a player with more resources and production may choose to throw attacks at the enemy regardless of the
+kill-to-casualty ratio in an effort to wear down the enemy, relying on the fact that they can afford to replace their
+losses but their opponent cannot. This is the equivalent of the real-world human wave attack.
+
+
+Area control. This refers not to the physical possession of a territory, but the ability to project influence over an area.
+A player who gains control over an area gains the initiative and the ability to make a wide range of tactical and
+
+strategic choices, giving him the power to maneuver the enemy and fight battles on his own terms.
+Fig. 3. Germany has just captured Anglo-Egypt. Germany has seized the initiative by gaining two great types of area control: 1) Control of the
+Suez Canal, which allows the Japanese fleet off the coast of India to join the German fleet in the Mediterranean. 2) Access to the lightlydefended interior of Africa, which Germany can quickly capture with its tanks and infantry at no cost.
+
+
+
+Area denial. Taking control of an area means the enemy correspondingly loses control there and is forced on the
+defensive. The enemy loses tactical flexibility and options and is forced to respond to your actions instead of taking
+initiative of his own.
+
+38
+
+Fig. 4. Germany has just seized Karelia with a large army and now stands facing a weaker Russian army in West Russia. Even if Russia
+counterattacked Karelia with everything that can reach it, he has almost no chance of destroying the stack of 11 German panzers and will suffer
+unacceptable casualties. If the Russian army stays in West Russia, it risks being annihilated on Germany’s next turn when Germany can bring to
+bear an overwhelming force of 11 infantry, 2 artillery, 13 tanks, 3 fighters and a bomber against West Russia. Russia has lost control of the area
+and is forced to retreat to either Caucasus or Russia or else suffer crushing defeat. Since the German move to Karelia denied Russia the ability to
+safely field any significant armies in the area, Germany can now trade and maneuver Archangel and West Russia at his leisure.
+
+
+Diversion. Often, diversionary tactics are needed to draw attention away from a hard-pressed ally or a collapsing
+front. These include amphibious landings and unexpected attacks along the periphery. Sometimes, an actual diversion
+is not needed—just the threat of a diversion suffices. For example, the threat of a D-Day invasion posed by a looming
+Anglo-American fleet in the Atlantic is often enough to cause the German player to divert units from the Eastern Front
+to defend France.
+
+Fig. 5. Japan makes a diversionary landing in Alaska to draw American attention away from Germany.
+
+39
+
+The Objectives of Defense
+There is a symmetry between the objectives of attack and those of defense. A player’s defensive tactics may be influenced by
+any one or combination of the following reasons:
+
+
```

Comment:
Those are some weird characters.
Looks like they are used to display non-standard characters

Reply:
I'm guessing it's the literal bulleted lists.

This iteration still marks the HTML as a WIP, but at least we can iterate on it more. So far this is just a few complete passes:
1. title links (some are actually broken)
2. images for the title sections and they are in place
3. images for the rule book.

We need to still take a couple more passes:
- clean up page numbers
- fix up line breaking
- see about getting some of the images to properly float left/right
- fix up table of content links
- consider updating anything that is out of date, perhaps pruning some items (EG: section on 'undo' buttons).

It seems as well the rule book has a bit of a identity crisis where it's both a how-to manual for how to play the game (mechanically), and it's a rule-book. We may want to consider splitting it up and/or moving the how-to pieces into the tutorial.